lighthouse-core/test/results/artifacts/defaultPass.trace.json

```diff
@@ -14505,10 +14505,8 @@
   {"pid":75994,"tid":17667,"ts":185608246435,"ph":"X","cat":"toplevel","name":"MessageLoop::RunTask","args":{"src_file":"../../ipc/ipc_mojo_bootstrap.cc","src_func":"SendMessage"},"dur":31,"tdur":30,"tts":78713},
   {"pid":75994,"tid":17667,"ts":185608246474,"ph":"X","cat":"toplevel","name":"MessageLoop::RunTask","args":{"src_file":"../../ipc/ipc_mojo_bootstrap.cc","src_func":"SendMessage"},"dur":16,"tdur":16,"tts":78752},
   {"pid":75994,"tid":17667,"ts":185608247189,"ph":"X","cat":"toplevel","name":"MessageLoop::RunTask","args":{"src_file":"../../ipc/ipc_mojo_bootstrap.cc","src_func":"SendMessage"},"dur":30,"tdur":29,"tts":78797},
-
-  {"_comment": "Manually added event to make sample lhr not error", "name":"largestContentfulPaint::Candidate", "pid":75994,"tid":17667,"ts":185608247190,"ph":"R","cat":"loading,rail,devtools.timeline", "args": {"frame": "0x44d2861df8"}},
-  {"_comment": "Manually added event to make test CLS", "name":"LayoutShift", "pid":75994,"tid":775,"ts":185608247190,"ph":"R","cat":"loading,rail,devtools.timeline", "args": {"frame": "0x44d2861df8", "data": {"is_main_frame": true, "cumulative_score": 0.42}}},
-
+  {"_comment":"Manually added event to make sample lhr not error","name":"largestContentfulPaint::Candidate","pid":75994,"tid":17667,"ts":185608247190,"ph":"R","cat":"loading,rail,devtools.timeline","args":{"frame":"0x44d2861df8"}},
```

Comment:
did something here change?

Reply:
huh strange looks like spaces have been removed in between the properties and values, maybe through one of those `yarn update` commands, I assume it's no biggie, the rest of the file appears to be consistent with that way of doing it

Reply:
can you do a `git checkout origin/master -- lighthouse-core/test/results/artifacts/defaultPass.trace.json` to revert, just so we don't lose blame here?

Reply:
Good point, thanks and done!

Reply:
> can you do a `git checkout origin/master -- lighthouse-core/test/results/artifacts/defaultPass.trace.json` to revert, just so we don't lose blame here?

FWIW @connorjclark these lines are going to have to lose their blame sooner or later :) I believe they were formatted by hand when added for one of the CLS audits, but as a result they get rewritten any time an automated tool touches the trace (since it's just json), and we shouldn't have to revert changes to it forever.